packages/icons/package.json

```diff
@@ -9,9 +9,20 @@
     "build": "rm -fr lib tsconfig.tsbuildinfo && tsc",
     "typecheck": "tsc --noEmit",
     "lint": "eslint ./src --ext .ts,.tsx --max-warnings 0",
-    "checks": "yarn typecheck && yarn lint"
+    "checks": "yarn typecheck && yarn lint",
+    "storybook:run": "start-storybook -p 6006",
+    "storybook:build": "build-storybook"
   },
   "devDependencies": {
+    "@storybook/addon-actions": "^6.5.6",
+    "@storybook/addon-essentials": "^6.5.6",
+    "@storybook/addon-interactions": "^6.5.12",
+    "@storybook/addon-links": "^6.5.6",
+    "@storybook/builder-webpack4": "^6.5.6",
+    "@storybook/jest": "^0.0.10",
+    "@storybook/manager-webpack4": "^6.5.6",
```

Comment:
Why webpack4? Maybe switch to vite?

Reply:
We need to generally rethink how we configure storybook. Vite would be great if that's possible. We also need to publish all stories from all packages somehow, which is a challenge.

@Andarist was looking into that, I can't find an issue for this though, lets treat it separately

Reply:
I copy pasted all storybook setup from another our package. We should update in all packages anyway if we're going to update.

Reply:
The proof of concept for that "root" storybook can be found [here](https://github.com/webstudio-is/webstudio-designer/pull/318). I need to figure out how to handle deployment and tooling stuff around this though.